testing/tests/DevExpress.ui.widgets.form/formLayoutManager.tests.js

```diff
@@ -2550,6 +2550,26 @@ QUnit.test("Change minColWidth when colCount is auto", function(assert) {
     invalidateStub.restore();
 });
 
+QUnit.test("Clear item watchers after disposing", function(assert) {
+    //arrange
+    var $testContainer = $("#container").width(450);
+
+    $testContainer.dxLayoutManager({
+        layoutData: { test1: "abc", test2: "qwe", test3: "xyz" },
+        colCount: 1,
```

Comment:
You can rid of `colCount` and `minColWidth`, it's not necessary to define it in this case